solidity/contracts/Coordinator.sol

```diff
@@ -329,15 +330,25 @@ contract Coordinator is ChainlinkRequestInterface, CoordinatorInterface {
    * @param _oracles is the list of oracle addresses
    */
   function aggregateAndPay(bytes32 _requestId, uint256 _paymentAmount, address[] memory _oracles) private returns (uint256) {
-    uint256 result;
+    uint divisor;
+    uint multiplier;
+    uint result;
```

Comment:
Just for consistency, can we use the full `uint256` type when declaring variables? I typically only use the `uint` alias in a for loop, but maybe we should be avoiding it there as well.

Reply:
Yes, I'll change these to uint256.